MultiMixEnv: Adds support for make

Tezirg opened the pull request.

Tezirg committed:
MultiMixEnv: Adds support for make

Tezirg committed:
Data dev: Removes some chronics in l2rpn_neurips_2020_track2

BDonnot commented on grid2op/MakeEnv/Make.py:
```diff
@@ -154,6 +159,17 @@ def _extract_ds_name(dataset_path):
 
+def _aux_is_multimix(dataset):
+    ds_name = os.path.basename(dataset)
+    if ds_name in MULTIMIX_ENVS:
```
not a fan at all of this. This re introduce a coupling between the env release and the dataset release

Tezirg replied:
Solved by using the meta file solution.

BDonnot reviewed (changes requested):
I think there are too much data in the data/l2rpn_neurips_2020_track2 (2 months per generation is sufficient imho).

Also, it would be nice to have something different to check if an env is a multimix or not. For example, we could set up a meta file in the env folder, if this file exist it specifies the env class, if not then Environment is used.
We could also imagine a file named like `__multimix` present in folder for multimix env and check whether or not this file is present in order to do the multimix constructor.

Tezirg committed:
MultiMix make support

Remove list a multimix env and use a dummy file, that indicates if env root folder is a multimix env

Tezirg commented:
Updated to use a metadata file instead of hardcoding multimix env names.
Dev data set `l2rpn_neurips_2020_track2` contains 2x2 chronics of 3 days for a total of 1.5Mb

Tezirg committed:
Dev data: l2rpn_neurips_2020_track2 removes some chronics

BDonnot commented on grid2op/MakeEnv/Make.py:
```diff
@@ -159,9 +155,10 @@ def _extract_ds_name(dataset_path):
+def _aux_is_multimix(dataset_path):
+    if os.path.exists(os.path.join(dataset_path, _MULTIMIX_FILE)):
         return True
     return False
```
`return os.path.exists(os.path.join(dataset_path, _MULTIMIX_FILE))` would probably be less confusing, but this works too ;-)
